server/src/main/scala/net/wiringbits/models/UserToken.scala

```diff
@@ -0,0 +1,15 @@
+package net.wiringbits.models
+
+import java.util.UUID
+
+case class UserToken(userId: UUID)
+
+object UserToken {
+  def validate(userToken: String): Option[UserToken] = {
+    try {
+      Some(UserToken(UUID.fromString(userToken)))
```

Comment:
`UUID.fromString` throws exceptions, you need `Try(UUID.fromString(...))`.map(...).toOption